### PR TITLE
FORMS-14622 AEM Adaptive forms: Panel Component issue with Radio Button component.

### PR DESCRIPTION
**Pull request by sakshi-arora1**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

File: it/content/src/main/content/jcr_root/content/dam/formsanddocuments/core-components-it/samples/radiobutton/radiorepeatability/_jcr_content/renditions/cq5dam.thumbnail.140.100.png.dir/.content.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (rismehta) [inline it/content/src/main/content/jcr_root/content/dam/formsanddocuments/core-components-it/samples/radiobutton/radiorepeatability/_jcr_content/renditions/cq5dam.thumbnail.140.100.png.dir/.content.xml:1]: Can you remove all these thumbnails

File: ui.tests/test-module/specs/prefill/customprefill.spec.js [hunk 2]
```diff
@@ -26,7 +26,7 @@ describe('Custom Prefill Test', function () {
     const pagePath = "content/forms/af/core-components-it/samples/prefill/basic.html";
     const nameTextBox = "input[name='name']",
           dobDropdown = "input[name='dob']",
-          genderRadioButton = "input[name='gender']",
+          genderRadioButton = "input[name='radiobutton-c8c660bac8_name']",
```
Comment (rismehta) [inline ui.tests/test-module/specs/prefill/customprefill.spec.js:29]: Why is this change required

**Review by rismehta — CHANGES_REQUESTED** [review]
minor comments

File: ui.tests/test-module/specs/prefill/customprefill.spec.js (hunk 2 above)
Comment (sakshi-arora1) [reply ui.tests/test-module/specs/prefill/customprefill.spec.js:29]: so that repeatability is supported. test case selector was written on old static behaviour

File: it/content/src/main/content/jcr_root/content/dam/formsanddocuments/core-components-it/samples/radiobutton/radiorepeatability/_jcr_content/renditions/cq5dam.thumbnail.140.100.png.dir/.content.xml (hunk 1 above)
Comment (sakshi-arora1) [reply it/content/src/main/content/jcr_root/content/dam/formsanddocuments/core-components-it/samples/radiobutton/radiorepeatability/_jcr_content/renditions/cq5dam.thumbnail.140.100.png.dir/.content.xml:1]: ok

Comment (sakshi-arora1) [reply it/content/src/main/content/jcr_root/content/dam/formsanddocuments/core-components-it/samples/radiobutton/radiorepeatability/_jcr_content/renditions/cq5dam.thumbnail.140.100.png.dir/.content.xml]: done

**Review by rismehta — APPROVED** [review]
